src/App.vue

```diff
@@ -597,26 +601,26 @@ export default {
       switch (status) {
         case FilingStatus.DRAFT:
         case FilingStatus.PENDING:
-          // this is a draft IA
-          this.setEntityStatus(EntityStatus.DRAFT_INCORP_APP)
-          this.storeDraftIa(ia)
+          // this is a draft application
+          this.setEntityStatus(EntityStatus.DRAFT_APP)
+          this.storeDraftAppTask(application)
```

Comment:
I'm so sorry you're in this code atm. Hopefully we can implement that other API ticket and get rid of this code.

Reply:
It's no problem! Things are pretty well laid out 👍